native/acter/src/api/settings/space.rs

```diff
@@ -182,9 +182,20 @@ impl Room {
 
         let client = self.room.client().clone();
         let room = self.room.clone();
+        let my_id = room
+            .client()
+            .user_id()
```

Comment:
```suggestion
        let my_id = client
            .user_id()
```

using it that often makes you wonder if we shouldn't just add that to `self`, no?

Reply:
Fixed by https://github.com/acterglobal/a3/pull/1671/commits/1a56d9a426d303deef8dfdcdb08c5c870f3a6716